### PR TITLE
harmony implementation code

### PR DESCRIPTION
**Pull request by PatExplores**
Hi Please merge the harmony code.

### CODE REVIEW

File: entry/src/main/java/com/kwabenaberko/openweathermap/slice/MainAbilitySlice.java [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+        helper.setUnits(Units.IMPERIAL);
+
+        //Set Languages
+        helper.setLanguage(Languages.ENGLISH);
+
+
+        new Thread(() -> helper.getCurrentWeatherByCityName("Accra", new CurrentWeatherCallback() {
+            @Override
+            public void onSuccess(CurrentWeather currentWeather) {
+                weatherReport = "Coordinates: " + currentWeather.getCoord().getLat() + ", "+currentWeather.getCoord().getLon() +"\n"
```
Comment (sharathks84) [inline entry/src/main/java/com/kwabenaberko/openweathermap/slice/MainAbilitySlice.java:99]: All the coordinates or results should be displayed in UI not on log.
If there is error and needs to refetch it should be handled.
UI should be added to get user information and also current information

**Comment by PatExplores** [comment]
demo app is improvised with city and gps weather options.

**Review by sharathks84 — APPROVED** [review]
ok, approved

**Review by Gleelakrishna — APPROVED** [review]
ok

**Pull request merged**